algoliasearch-core/src/main/java/com/algolia/search/HttpTransport.java

```diff
@@ -27,7 +27,7 @@
   private final HttpRequester httpRequester;
   private final RetryStrategy retryStrategy;
   private final ConfigBase config;
-  private static final Logger logger = LoggerFactory.getLogger("algoliasearch");
+  private static final Logger LOGGER = Logger.getLogger(HttpTransport.class.getName());
```

Comment:
For another commit: ensure all static-final are uppercase:
- make javaVersion uppercase in Config.base
- if possible add a rule in the formatting configuration